webapp/WEB-INF/templates/admin/plugins/newsletter/create_newsletter.html

```diff
@@ -18,7 +18,7 @@
 							<@inputGroupItem type='text'>
 								<@icon style='calendar' />
 							</@inputGroupItem>
-							<@input type='date' name='date_first_send' id='date_first_send' format='j/m/Y' maxlength=10 value='${newsletter_last_sent}' />
+							<@input type='date' name='date_first_send' id='date_first_send' maxlength=10 value='${newsletter_last_sent}' />
```

Comment:
to avoid internationalisation problems, the default date should be sent as a date object : 
<@input type='date' name='date_first_send' id='date_first_send' maxlength=10 value=theDefaultDateObject />